sound/soc/codecs/rt715-sdca.c

```diff
@@ -268,6 +269,7 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 	unsigned int reg2 = mc->rreg;
 	unsigned int reg = mc->reg;
 	unsigned int max = mc->max;
+	unsigned int val0, val1;
```

Comment:
`struct rt715_sdca_priv *rt715 = snd_soc_component_get_drvdata(component);` is required, otherwise, it can't build.

Reply:
will fix it in next v3 patch.